core/services/ocr2/plugins/ocr2keeper/evm20/log_provider.go

```diff
@@ -18,8 +18,6 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/logpoller"
 	registry "github.com/smartcontractkit/chainlink/v2/core/gethwrappers/generated/keeper_registry_wrapper2_0"
 	"github.com/smartcontractkit/chainlink/v2/core/logger"
-	"github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ocr2keeper/evm"
-	pluginevm "github.com/smartcontractkit/chainlink/v2/core/services/ocr2/plugins/ocr2keeper/evm"
```

Comment:
just confirming @EasterTheBunny , plugin does not contain an evm package anymore?

Reply:
I moved log_provider into the evm folder, therefore the import is not needed

Reply:
oh, looking at it more closely they were both referencing the exact same files :sweat_smile: . I thought there were separate 'evm' packages in core and plugin but seems like that's not the case anymore

Reply:
Right. I have tried to remove all evm stuff from the plugin. It should all be in the core node now.